core/src/main/scala/kafka/server/AdminManager.scala

```diff
@@ -473,6 +479,7 @@ class AdminManager(val config: KafkaConfig,
       alterConfigOp.opType() match {
         case OpType.SET => Log4jController.logLevel(loggerName, logLevel)
         case OpType.DELETE => Log4jController.unsetLogLevel(loggerName)
+        case _ =>
```

Comment:
Good fix. Perhaps we should raise an `IllegalArgumentException` or something in this case?

Reply:
Done.